Macros/e3 Includes/e3_Classes_Shaman.inc

```diff
@@ -77,6 +81,24 @@ sub Event_AE_AA(string line)
 	}
 /return
 |----------------------------------------------------------------------------|
+#event Idol "[MQ2] Idol"
+#event Idol "#1# tells you, 'Idol'"
+#event Idol "Soul Idol has expired."
+SUB Event_Idol(line, ChatSender)
+	/tar Soul Idol
+	/delay 1s
+	/if (${Target.CleanName.Equal[Soul Idol]}) { 
+		/g Unable to summon idol. One is already active
+		/return
+	}
```

Comment:
This will check for soul idol without changing your target: (!${Bool[${Spawn[Soul Idol]}]})

Reply:
Much cleaner. I like it